pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -1146,12 +1145,11 @@ func LoadIPSecKeys(log *slog.Logger, r io.Reader) (int, uint8, error) {
 		ipSecKey.Spi = spi
 		ipSecKey.ESN = esn
 
-		if ipSecKeysGlobal[""] != nil {
-			oldSpi = ipSecKeysGlobal[""].Spi
+		if oldKey, ok := ipSecKeysGlobal[""]; ok {
+			// TODO(smagnani96) add checks to return error if unchanged SPI
```

Comment:
Yes, if it's important enough we should just fix it. If we really think it's not important then it won't get done just because we add a "TODO" comment.

> _Do, or do not. There is no try._
> Yoda

Reply:
Joe: For context, there's another PR opened to fix this.

Reply:
Thanks both for the clarification, got the message :muscle: 
Removed right now.